### ISSUE TITLE
Locale errors—in 2020!

### ISSUE THREAD

**Issue by yadayadaydadaa** [mrcrowl/vscode-hg#80]
After upgrading to Mercurial 5.4 and Python 3, I get the following on macOS:

```
hg root
Fatal Python error: config_get_locale_encoding: failed to get the locale encoding: nl_langinfo(CODESET) failed
Python runtime state: preinitialized
```

I have `LANG=en_GB.UTF-8` in my `~/.profile` and Python 3/Mercurial work from the command line, but not from VSCode.

**Comment by yadayadaydadaa** [comment]
I think this has something to do with the `LC_CTYPE` setting. On the command line `LC_CTYPE=en_GB.UTF-8`  and `LC_CTYPE=UTF-8` work, but `LC_CTYPE=en_GB` causes the the error. In the terminal, by default, without anything having been touched, the middle one holds. So it seems either the extension is touching `LC_CTYPE`, or something in in the way VSCode is loading the environment (doesn't matter where it's started from) messes up `LC_CTYPE`.

**Comment by yadayadaydadaa** [comment]
I tried to create a wrapper script, to check the environment, but `hg.path` does not seem to be respected.

**Comment by incidentist** [comment]
Fixed in 1.4.0.
